bundles/SimpleBackendSearchBundle/src/Controller/SearchController.php

```diff
@@ -143,7 +143,11 @@ public function findAction(Request $request, EventDispatcherInterface $eventDisp
                 //this loop divides filter parameters to localized and unlocalized groups
                 $definitionExists = in_array($paramConditionObject['property'], DataObject\Service::getSystemFields())
                     || $class->getFieldDefinition($paramConditionObject['property']);
-                if ($definitionExists) { //TODO: for sure, we can add additional condition like getLocalizedFieldDefinition()->getFieldDefiniton(...
+
+                if (
+                    $definitionExists
+                    && !$class->getFieldDefinitions()['localizedfields']->getFieldDefinition($paramConditionObject['property'])
```

Comment:
@saschabaecher-twocream Can you test it with a class that does not have a localized field?

I think there is no localizedfields array key and an exception will occur.

Reply:
Hello @blankse, thank you for your feedback!

If there are no localized fields in the class, an error occurs. I have now adjusted this.